src/Strategy/Gradle.hs

```diff
@@ -94,7 +94,7 @@ analyze BasicDirOpts{..} =
 
     let text = decodeUtf8 $ BL.toStrict stdout
         textLines :: [Text]
-        textLines = T.lines text
+        textLines = T.lines (T.filter (/= '\r') text)
```

Comment:
We should add a test for this by duplicating the current test file and changing the endings.

Reply:
I'll follow up today with another PR for normalizing output from Exec/ReadFS. There's another case we need to properly handle too -- and that's when Exec/ReadFS outputs aren't UTF8